fairseq/training.py

```diff
@@ -38,6 +38,13 @@ def __init__(
         self.package_name = package_name
         self.check_consistency()
 
+    @property
+    def data(self):
+        """
+        Get the underlying data of the FairseqHydraConfig
+        """
+        return self.config_dict
+
```

Comment:
I guess this is not really needed. Do you think it improves the code quality/readability?

Reply:
In my opinion, accessing the actual config data in the `fairseq_hydra_config` with `fairseq_hydra_config.data` is more readable and clearer than `fairseq_hydra_config.config_dict`. However, it is not strictly necessary and would still work with the latter option, so in the end its just up to taste